src/rules/export.js

```diff
@@ -50,7 +50,13 @@ module.exports = function (context) {
         return
       }
       let any = false
-      remoteExports.forEach((v, name) => (any = true) && addNamed(name, node))
+      remoteExports.forEach((v, name) => (
+        (any = true) &&
```

Comment:
I know it was already in the original code (so this is more like a question to @benmosher), but wouldn't it be simpler & less odd (an assignation in a condition? oô), to do `const any = remoteExports.length > 0`?

I would actually split it up somewhat like this
```js
const namedExports = remoteExports.filter((v, name) => name !== 'default');
const anyNamed = namedExports.length > 0;
namedExports.forEach((v, name) => addNamed(name, node));
```